src/Blorc.OpenIdConnect/Extensions/JsonElementExtensions.cs

```diff
@@ -53,8 +56,11 @@ public static IEnumerable<Claim> AsClaims(this JsonElement element, string claim
                     break;
 
                 case JsonValueKind.True:
+                    yield return new Claim(claimType, TrueAsString);
+                    break;
+
                 case JsonValueKind.False:
-                    yield return new Claim(claimType, element.GetBoolean().ToString());
+                    yield return new Claim(claimType, FalseAsString);
```

Comment:
Great optimization! I think there is no need to store them in fields though. Is there any argument against using bool.FalseString here?